File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        store ={}

        for num in nums:
            store[num] = store.get(num,0)+1
            
            
        freqlist = []
        for _ in range(len(nums)+1):
            freqlist.append([])
        for num , freq in store.items():
            freqlist[freq].append(num)
        res=[]
        for x in range(len(nums),0,-1):
            for num in freqlist[x]:
                res.append(num)
            if len(res)==k:
                return res
                       
        

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
#include<unordered_map>
#include<vector>
using namespace std;
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int,int>store;
        for(size_t i=0;i<nums.size();i++){
            if(store.find(nums[i])!=store.end()){
                return true;
            }
            store[nums[i]]=i;
            

        }
        return false;
    }
};